.github/README.md

```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
 ```json
 {
   "transcription": {
-    "spa": "hola me llamo Daniel",
+    "sp": "hola me llamo Daniel",
     "ipa": "ola me jamo dænjəl"
```

Comment:
Now that I see this change, it occurs to me we should also change the IPA transcription to `spa-fonipa` (which is the official language tag for transcribing Spanish in IPA).

```suggestion
    "es-fonipa": "ola me jamo dænjəl"
```